ocp_resources/utils.py

```diff
@@ -1,229 +1,7 @@
-import datetime
-import time
-from warnings import warn
-
-
 import yaml
 from simple_logger.logger import get_logger
 
 LOGGER = get_logger(name=__name__)
-DEPRECATED_MSG = (
```

Comment:
Please do not remove the timeout sampler functionality from 4.15 branch.

Leave it as is, we may have samo users using it

Reply:
Sure, thanks, left it as is